### ISSUE TITLE
Unable to return IfThenElse expression from a Method

### ISSUE THREAD

**Issue by JordanMarr** [edgarfgp/Fabulous.AST#69]
Things are flying along with my rewrite at a fast pace.
However, one issue I am having is that I am unable to return an `IfThenElse` widget from a `Method`.

```F#
    Method("__.DoSomething",
        UnitPat(),

        IfThenElse(
            InfixAppExpr(ConstantExpr(Constant("x", false)), "=", ConstantExpr(Constant("12", false))),
            ConstantExpr(ConstantUnit()),
            ConstantExpr(ConstantUnit())
        )

    )
```
"The type `WidgetBuilder<ExprIfThenElseNode>` is not compatible with type `WidgetBuilder<Expr>`."

**Comment by edgarfgp** [comment]
Yeah. This is a bug. Let me have a look

**Comment by edgarfgp** [comment]
Fixed in #70. Should be available in `0.9.0`

```fsharp
Method(
      "__.DoSomething",
      UnitPat(),
      IfThenElseExpr(
          InfixAppExpr(ConstantExpr(Constant("x", false)), "=", ConstantExpr(Constant("12", false))),
          ConstantExpr(ConstantUnit()),
          ConstantExpr(ConstantUnit())
      )

  )
```
